app/services/publisher/service_provisioner.rb

```diff
@@ -130,7 +130,7 @@ def live_production?
     end
 
     def config_map
-      service_configuration.reject(&:secrets?).reject(&:do_not_send_submission?).reject(&:do_not_send_confirmation_email?)
+      service_configuration.reject(&:secrets?).reject(&:do_not_send_submission?).reject(&:do_not_send_confirmation_email?).reject(&:not_in_maintenance_mode?)
```

Comment:
```
To everything chain, chain, chain
There is a season chain, chain, chain
```

i feel this is ripe for a refactor. as we have found between the bug fix and your story the logic of it is not entirely easy to parse. 

i guess in another PR though

Reply:
yeah probably.  Its definitely not the most readable line of code! I guess for a refactor we'd likely want to push more of it back into the `ServiceConfiguration`? But definitely for another PR